web/packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -186,6 +189,21 @@ export class WorkspacesService extends ImmutableStore<WorkspacesState> {
     this.persistState();
   }
 
+  setConnectMyComputerAutoStartFlag(
```

Comment:
What do you think about an API such as this?

```typescript
setConnectMyComputerAutoStart(rootClusterUri: RootClusterUri, autoStart: boolean)
```

`Flag` in the name seems a bit too verbose in my opinion. I don't get the advantage of the `params` argument, I guess it was to avoid having a bare boolean argument at the callsites? I think that's fine for simple setters tbh.

If we drop `Flag` from this name, we should also drop it from the getter.

Reply:
Nice, done. 
>I don't get the advantage of the params argument, I guess it was to avoid having a bare boolean argument at the callsites?

Yeah, that was my aim, but getting rid of this is fine.